cypress/e2e/supervision/finance/client-finance-tab.cy.js

```diff
@@ -45,7 +45,7 @@
       cy.get('#add-finance-discount-button', { timeout: 2000 }).should('be.visible');
       cy.get('#add-finance-discount-button').click();
       cy.get('.head > .title').should('contain.text', 'Award fee reduction');
-      cy.get(':nth-child(1) > .radio-button').click();
+      cy.contains('Remission').click();
```

Comment:
This'll pick up any use of the term Remission on the page and click that, which might be an earlier element than the radio you want, so might make the test flakey as the UI changes.